apps/dashboard-api/domain/property/model.go

```diff
@@ -28,13 +28,13 @@ type SingleFamilyProperty struct {
 	Unit          SingleFamilyPropertyUnit `json:"unit,omitempty"`
 }
 
-func (SingleFamilyProperty) IsProperty()     {}
-func (SingleFamilyProperty) IsPropertyInfo() {}
+func (*SingleFamilyProperty) IsProperty()     {}
```

Comment:
To match the generated code and it requires fewer value <> pointer mapping (simpler).